src/OSPSuite.SimModel/managed/src/Conversions.cpp

```diff
@@ -54,7 +54,7 @@ namespace SimModelNET
 
 	array<double>^ CPPToNETConversions::DoubleVectorToArray(const std::vector<double> & doubleVec)
 	{
-		array<double>^ doubleArray = gcnew array<double>(doubleVec.size());
+		array<double>^ doubleArray = gcnew array<double>((unsigned int)doubleVec.size());
 
```

Comment:
Explicit conversion to avoid compiler warning. Same in the most code changes below